core-services/prow/02_config/_config.yaml

```diff
@@ -354,7 +354,7 @@ plank:
         mediaTypes:
           log: text/plain
       gcs_credentials_secret: gce-sa-credentials-gcs-qe-private-deck
-      timeout: 22h0m0s
+      timeout: 23h0m0s
```

Comment:
I don't think we should change this at the moment, since it's still not clear why IBM post stage only take 1 hour and timeout while it's given two hours for timeout.

Reply:
1h is a grace period timeout